components/table/table/tableFilters/TableFilters.module.scss

```diff
@@ -1,5 +1,85 @@
 @import '../..../../../../../styles/';
 
+.v2 {
+	@include flex(row, space-between, center);
+	padding: 1.25rem;
+
+	.details {
+		@include flex(column, center, flex-start);
+		gap: 0.5rem;
+		width: 30%;
```

Comment:
Don't give width in constant static percentage values. Always use flex and adjust accordingly if padding is needed.